File: Lista2/C/Lsala2/Ex8.c
#include <stdio.h> 

int main()
{
	int n1, n2, n3;
	printf("1o Numero: ");
	scanf("%i", &n1);
	printf("2o Numero: ");
	scanf("%i", &n2);
	printf("3o Numero: ");
	scanf("%i", &n3);
	if(n1%2==0 && n1%3==0 || n2%2==0 && n2%3==0 || n3%2==0 && n3%3==0){
		if(n1%2==0 && n1%3==0){
			printf("O número %i é divesível por 2 e por 3", n1);
		}
		if(n2%2==0 && n2%3==0){
			printf("O número %i é divesível por 2 e por 3", n2);
		}
		if(n3%2==0 && n3%3==0){
			printf("O número %i é divesível por 2 e por 3", n3);
		}
	}else{
		printf("Nenhum dos numeros e divisivel por 2 e por 3");
	}
}

File: Lista2/Python/Lsala2/Ex8.py
n1 = int(input("1º Número: "))
n2 = int(input("2º Número: "))
n3 = int(input("3º Número: "))
if n1 % 2 == 0 and n1 % 3 == 0 or n2% 2 == 0 and n2 % 3 == 0 or n3 % 2 == 0 and n3 % 3 == 0:
    if n1 % 2 == 0 and n1 % 3 == 0:
        print(f"O número {n1} é divesível por 2 e por 3")
    if n2 % 2 == 0 and n2 % 3 == 0:
        print(f"O número {n2} é divesível por 2 e por 3")
    if n3 % 2 == 0 and n3 % 3 == 0:
        print(f"O número {n3} é divesível por 2 e por 3")
else:
    print("Nenhum dos números é divisível por 2 e por 3")
